packages/devextreme/js/core/utils/object.js

```diff
@@ -39,16 +39,29 @@ const orderEach = function(map, func) {
     }
 };
 
-const assignValueToProperty = function(target, property, value, assignByReference) {
+const assignValueToProperty = function(target, property, value, extendComplexObject, assignByReference, shouldCopyUndefined) {
```

Comment:
Question.

Maybe adding the new argument at the end of the argument list will be better?
Now in the old code the ```assignByReference: true``` became the ```extendComplexObject: true, assignByReference: undefined```

Reply:
Strange diff here
As you can see, at the [branch 24_2](https://github.com/DevExpress/DevExtreme/blob/24_2/packages/devextreme/js/core/utils/object.js#L51), this flag is already present